packages/react-native/Libraries/Text/TextInput/Multiline/RCTUITextView.mm

```diff
@@ -13,6 +13,33 @@
 #import <React/RCTBackedTextInputDelegateAdapter.h>
 #import <React/RCTTextAttributes.h>
 
+//the UITextSelectionRect subclass needs to be created because the original version is not writable
+@interface CustomTextSelectionRect : UITextSelectionRect
```

Comment:
UITextSelectionRect is explicitly documented as being an abstract base class so this makes sense. Maybe mention that instead of it not being writable?

Use `RCT` as prefix

```suggestion
@interface RCTTextSelectionRect : UITextSelectionRect
```

Reply:
good point, changed